person_spec.rb

```diff
@@ -0,0 +1,8 @@
+require './person'
+describe Person do
```

Comment:
- [ ] We are expected to kindly covered all the scenarios and functionalities of the Person class:
- Let's kindly test the situation were the constructor method sets a default value for the optional `name` parameter if the name is not provided by the user. **Hints:** `expect(person.name).to eql "Unknown"` :+1: 
- Based on the requirements of this project, we are expected to tests all methods in all classes. Let's kindly add tests for the `add_rental` method
- In our app, we are expected to provide some level of permission when creating a person object(student) hence let's kindly tests both positive and negative scenarios of the `can_use_services?` method :+1: 
- We equally got another method in the Person class that really assist us to trim the person's name to 10 characters. I think this awesome functionality deserves to be tested :+1:  What do you think?